### ISSUE TITLE
Puzzle piece graphic in dev hub banner is slightly mis-rotated

### ISSUE THREAD

**Issue by atsay** [mozilla/addons-frontend#1715]
The gray puzzle piece in the dev hub banner is not perfectly square. It seems to be a bit tilted to the left. @potch spotted it.

https://www.dropbox.com/s/2zp91ynq4or129f/Screenshot%202017-02-06%2013.10.10.png?dl=0

**Comment by pwalm** [comment]
Yeah, this is an issue with the icon itself, as it was designed not symmetrical. It's been on my to-do list for awhile, but it is like a P5.

**Comment by atsay** [comment]
@potch can you attach your svg here for @pwalm to take a look? (It might help?)

**Comment by potch** [comment]
I took the existing vector and made minor tweaks, then cut it in half and mirrored it so it should be perfectly symmetrical :) 
![addons-puzzle](https://cloud.githubusercontent.com/assets/253470/22711902/af750208-ed37-11e6-99f2-f28399098320.png)

The SVG of said image is here: http://bits.potch.me/addons-puzzle.svg

**Comment by pwalm** [comment]
You handsome genius! Thank you for doing that. 😄
